### PR TITLE
Mlr 25

### PR DESCRIPTION
**Pull request by ayan-usgs**
@isuftin -- Not perfect, but I think it will suffice as the Python component of the POC.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,6 +255,21 @@ While the Docker registry container is known to other containers using the netwo
 
 - `$ docker-compose up registry`
 
+## PyPI
+
+Artifactory OSS does not support PyPI repositories. In order to store Python artifacts, a separate PyPI container is included.
+
+## The PyPI Secrets Directory
+
+A user and password must be setup in the PyPI repo before artifacts can be deployed to it. If there is not already a `secrets`
```
Comment (isuftin) [inline README.md]: Are you able to include a default for this as I did with other projects within this repo? This saves the end user from having to do this themselves and speeds the setup of the project.

File: README.md [hunk 2]
```diff
@@ -284,6 +299,19 @@ keystore so that the Jenkins instance may communicate with HTTPS websites
 in GitHub to provide access to the repositories that include your microservice stacks.
 See [GitHub documentation](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) for more information
 
+- `.pypirc`: This is a configuration file that tells `pip` how to upload Python artifacts to
```
Comment (isuftin) [inline README.md]: Are you also able to include the default version of this file in the source in this repo?

Comment (ayan-usgs) [inline README.md]: Yes, I can include a default version of both the `.htaccess` and `.pypirc` files.

Comment (isuftin) [inline README.md]: @ayan-usgs Excellent. I think once those two files are in place, this should build without a user having to further configure anything for the pypi repo server, is that correct? Should just be `docker-compose up pypi` and that's it?

Comment (ayan-usgs) [inline README.md]: Yep.

**Comment by ayan-usgs** [comment]
@isuftin -- made requested changes.

**Comment by isuftin** [comment]
@ayan-usgs Now that you have .htpasswd in that dir, you can probably remove .keep

Other than that, you should be 👍

File: .gitignore [hunk 3]
```diff
@@ -1,9 +1,13 @@
 .DS_Store
 !.keep
+!./pypi/secrets/.htpasswd
```
Comment (isuftin) [inline .gitignore]: This is probably not needed anymore?

**Pull request merged**